packages/core/src/di/PluginLoader.ts

```diff
@@ -27,6 +32,10 @@ export class PluginLoader implements PluginResolver {
     });
   }
 
+  public resolveValidationSchemaContributions(): object[] {
```

Comment:
Why don't you use a getter here?

Reply:
Yeah, I guess we could. Wanted to keep it in line with the other method. Also want to keep it open for us to do a dynamic search in the future.